plugin/grpc/grpcstats/client_metrics.go

```diff
@@ -42,24 +42,6 @@ var (
 	RPCClientResponseBytesView    *stats.View
 	RPCClientRequestCountView     *stats.View
 	RPCClientResponseCountView    *stats.View
-
-	RPCClientRoundTripLatencyMinuteView *stats.View
```

Comment:
Hmm, let's also coordinate removing these views and interval windows from the Java client /cc @songy23 @sebright

Reply:
I would prefer not to block this PR on other languages. Interval views are currently broken in Go anyway.

Reply:
/cc @dinooliva